octobot_trading/producers/ohlcv_updater.py

```diff
@@ -83,8 +84,15 @@ async def _initialize_candles(self, time_frame, pair):
         candles: list = await self.channel.exchange_manager.exchange \
             .get_symbol_prices(pair, time_frame, limit=self.OHLCV_OLD_LIMIT)
         self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
-        await self.channel.exchange_manager.get_symbol_data(pair) \
-            .handle_candles_update(time_frame, candles[:-1], replace_all=True, partial=False)
+        if candles:
+            await self.channel.exchange_manager.get_symbol_data(pair) \
+                .handle_candles_update(time_frame, candles[:-1], replace_all=True, partial=False)
+        else:
+            # When candle history cannot be loaded, retry to load it later
+            self.logger.error(f"Failed to init candle history for {self.channel.exchange_manager.exchange_name}")
+            asyncio.get_event_loop().call_later(self.OHLCV_INITIALIZATION_RETRY_DELAY,
+                                                asyncio.create_task,
+                                                self._initialize_candles(time_frame, pair))
```

Comment:
:+1:
I think we should put this in a while with like 10s delay, this is a very important data.